packages/client/components/ActivityLibrary/CreateActivityCard.tsx

```diff
@@ -1,72 +1,35 @@
 import React from 'react'
-import graphql from 'babel-plugin-relay/macro'
 import {CategoryID} from './ActivityCard'
-import {ActivityLibraryCard, ActivityLibraryCardBadge} from './ActivityLibraryCard'
+import {ActivityLibraryCard} from './ActivityLibraryCard'
+import {ActivityBadge} from './ActivityBadge'
 import {Add as AddIcon} from '@mui/icons-material'
 import {CATEGORY_ID_TO_NAME} from './ActivityLibrary'
 import clsx from 'clsx'
-import useModal from '../../hooks/useModal'
-import {useFragment} from 'react-relay'
-import {CreateActivityCard_teams$key} from '~/__generated__/CreateActivityCard_teams.graphql'
-import {CreateActivityCard_templates$key} from '~/__generated__/CreateActivityCard_templates.graphql'
-import TeamPickerModal from './TeamPickerModal'
+import {Link} from 'react-router-dom'
 
 interface Props {
-  category: CategoryID
   className?: string
-  teamsRef: CreateActivityCard_teams$key
-  templatesRef: CreateActivityCard_templates$key
+  category: CategoryID
 }
 
 const CreateActivityCard = (props: Props) => {
-  const {category, className, teamsRef, templatesRef} = props
```

Comment:
this component was heavily simplified due to unified navigation - ie. clicking on create custom activity always links to create new activity screen